src/components/pages/home/hero/hero.jsx

```diff
@@ -1,159 +1,124 @@
-            />
-          </div>
-
-          <div className="absolute right-[81px] top-[89px] h-6 w-6 3xl:right-[81px] 3xl:top-[67px] 2xl:right-[68px] 2xl:top-[2px] xl:hidden lg:right-[47px] lg:top-[1px] lg:block sm:hidden">
-            <div className="absolute left-0 top-0 h-6 w-12 rounded-l-full bg-black" />
-            <div
-              className="circle circle-with-text circle-with-text-right !translate-x-0 !translate-y-0"
-              data-text="Hasura Cloud"
-            />
-          </div>
-
-          <div className="absolute left-[-5px] top-[185px] h-6 w-6 3xl:left-[4px] 3xl:top-[152px] 2xl:left-[-3px] 2xl:top-[128px] xl:left-[-9px] xl:top-[54px] lg:left-[-3px] lg:top-[97px] sm:hidden">
-            <div className="absolute right-0 top-0 h-6 w-12 rounded-r-full bg-black" />
-            <div
-              className="circle circle-with-text circle-with-text-left !translate-x-0 !translate-y-0"
-              data-text="Vercel"
-            />
-          </div>
-
-          <div className="absolute bottom-[61px] left-[41px] top-auto h-6 w-6 3xl:left-[39px] 2xl:bottom-[51px] 2xl:left-[25px] xl:bottom-[44px] xl:left-[15px] lg:bottom-[36px] lg:left-[17px] sm:hidden">
-            <div className="absolute right-0 top-0 h-6 w-12 rounded-r-full bg-black" />
-            <div
-              className="circle circle-with-text circle-with-text-left !translate-x-0 !translate-y-0"
-              data-text="Netlify"
-            />
-          </div>
+          Serverless Postgres
+        </Heading>
+        <p className="t-xl mx-auto mt-7 max-w-[860px] text-white 2xl:mt-6 xl:mt-5">
+          The fully managed serverless Postgres with a generous free tier. We separate storage and
+          compute to offer autoscaling, branching, and bottomless storage.
+        </p>
+        <div className="mx-auto mt-11 grid max-w-[452px] auto-rows-fr grid-cols-2 items-center gap-x-6 2xl:mt-8 xl:mt-7 md:mt-6 xs:grid-cols-1 xs:gap-y-4">
+          {buttons.map(({ text, to, theme, className }) => (
+            <Button
+              key={text}
+              className={clsx(className, 'h-20 2xl:h-[72px] xl:h-[60px] md:h-14')}
+              to={to}
+              theme={theme}
+              size="md"
+            >
+              {text}
+            </Button>
+          ))}
+        </div>
+      </div>
 
-          <div className="absolute bottom-[-48px] right-[233px] top-auto h-6 w-6 3xl:bottom-[-28px] 3xl:right-[190px] 2xl:bottom-[-29px] 2xl:right-[162px] xl:hidden lg:right-[131px] lg:block sm:hidden">
-            <div className="absolute left-0 top-0 h-6 w-12 rounded-l-full bg-black" />
-            <div
-              className="circle circle-with-text circle-with-text-right !translate-x-0 !translate-y-0"
-              data-text="Amazon Lambda"
+      <ul className="mx-auto mt-20 flex w-full max-w-[1472px] justify-between gap-x-16 xl:gap-x-14 lg:mt-16 lg:flex-wrap lg:justify-center lg:gap-y-10 md:mt-14 md:gap-y-8 sm:mt-10 xs:gap-y-6">
```

Comment:
I think we need to tweak two things:

1. Add a headline above the logos
2. Make the logos and headline less visually significant than the rest of the hero (I did it with opacity, you can do it some other way)

```suggestion
      <Heading
          id="case-studies-title"
          tag="h3"
          className="t-xl text-white mt-20 font-bold opacity-75"
        >
          Powering 500,000+ databases for developers and partners
        </Heading>
      <ul className="mx-auto mt-12 flex w-full max-w-[1472px] justify-between gap-x-16 xl:gap-x-14 lg:mt-16 lg:flex-wrap lg:justify-center lg:gap-y-10 md:mt-14 md:gap-y-8 sm:mt-10 xs:gap-y-6 opacity-75">
```